### DISCUSSION TITLE
Local Installation Failed due to storage issue

### DISCUSSION THREAD

**Discussion by Tulasi08** [broadinstitute/seqr#3808] [Q&A]
Hello Team,

We are installing seqr locally and it is mentioned that the prerequisite is "Hardware: At least 16 Gb RAM, 4 CPUs, 50 Gb disk space" but during the installation and when we use "Option #2: annotate and load on-prem" and ran the command "docker-compose exec pipeline-runner download_reference_data.sh $BUILD_VERSION" we have encountered an error:
 
"Caught ResumableDownloadException (Transfer failed after 23 retries. Final exception: b'[Errno 28] No space left on device') for download of ./combined_reference_data_grch37.ht/rows/parts/part-20867-0-20867-0-0b7c07d6-1d93-cd26-094f-e6f2738b05a2.
ResumableDownloadException: Transfer failed after 23 retries. Final exception: b'[Errno 28] No space left on device'
Copying gs://seqr-reference-data/GRCh37/all_reference_data/combined_reference_data_grch37.ht/rows/parts/part-20889-0-20889-0-edc76f29-2f8a-1c73-2ecb-0845a8236b91..."

While looking into the size of the /data/seqr-reference-data/GRCh37/combined_reference_data_grch37.ht in our machine we can see that 188 GB is downloaded. 

Kindly requesting you to tell us how much more disk space would be required to install seqr locally while using option 2 to annotate because we can tell our IT team accordingly. 

![image](https://github.com/broadinstitute/seqr/assets/122073339/91ee39a6-aae9-475c-8c47-ec2bc6204395)

![image](https://github.com/broadinstitute/seqr/assets/122073339/ade789ac-7368-4231-8ac1-35abd491707b)

Also could you please help me on how to resume this download after we have increased our disk space without starting the download from the beginning like without the the VEP download?

**Comment by hanars** [comment]
The minimal hardware requirement specified in the readme is a lower bound for running seqr, and it assumes a very minimal seqr installation using the lowest memory usage for data loading and a very small amount of total data. Those estimates were also designed to let you know whether or not a single laptop would work for you to run seqr directly, not as guidance for an IT team installing seqr on some sort of scalabe architecture.  

We do not provide upper bound estimates or more detailed hardware requirements because seqr's resource usage varies wildly depending on how much data you plan to load, whether that data is WES or WGS, whether you need to support both GRCh37 and GRCh38 etc. We do not really know how big various local installations end up being, due to all this variability, so this is unfortunately not something we can give guidance on.

If your IT team has the ability to arbitrarily resize the avaible disk space as you imply they do, my advice to them would be that they over-provision it for the initial installation, and then after the install and data loading completes they calculate the actual disk usage and adjust back down

**Reply by Tulasi08** [reply]
Thank you for the clarification. Will convey this to our IT team!

Also could you please help me on how to resume this download after we have increased our disk space without starting the download from the beginning like without the the VEP download?

**Reply by hanars** [reply]
I would strongly recommend you start a clean installation on new hardware, trying to resume a partial installation is likely to end up with bugs

**Reply by Tulasi08** [reply]
Sure will do that! Thanks again!
